string goal_waypoint
---
bool success
---
string feedback
